examples/tinywl/StackToplevelHelper.qml

```diff
@@ -20,24 +20,69 @@ Item {
     property bool mapped: waylandSurface.surface && waylandSurface.surface.mapped && waylandSurface.WaylandSocket.rootSocket.enabled
     property bool pendingDestroy: false
     property bool isMaximize: waylandSurface && waylandSurface.isMaximized && outputCoordMapper
-
+    property bool isFullScreen: waylandSurface && waylandSurface.isFullScreen && outputCoordMapper
+/*
     Binding {
         target: surface
         property: "states"
         restoreMode: Binding.RestoreNone
-        value: State {
-            name: "maximize"
-            when: isMaximize
-            PropertyChanges {
-                restoreEntryValues: true
-                target: root.surface
-
-                x: outputCoordMapper.x + Helper.getLeftExclusiveMargin(waylandSurface)
-                y: outputCoordMapper.y + output.topMargin + Helper.getTopExclusiveMargin(waylandSurface)
-                width: outputCoordMapper.width - Helper.getLeftExclusiveMargin(waylandSurface) - Helper.getRightExclusiveMargin(waylandSurface)
-                height: outputCoordMapper.height - output.topMargin - Helper.getTopExclusiveMargin(waylandSurface) - Helper.getBottomExclusiveMargin(waylandSurface)
+        value: [
+            State {
```

Comment:
qrc:/qt/qml/Tinywl/StackToplevelHelper.qml:30:13: Cannot assign multiple values to a singular property

@zccrs Should we move this to StackWorkspace?